deisctl/cmd/upgrade.go

```diff
@@ -65,16 +69,15 @@ func republishServices(ttl uint64, nodes []*model.ConfigNode, cb config.Backend)
 }
 
 // UpgradeTakeover gracefully starts a platform stopped with UpgradePrep
-func UpgradeTakeover(b backend.Backend, cb config.Backend) error {
-
-	if err := doUpgradeTakeOver(b, cb); err != nil {
+func UpgradeTakeover(stateless bool, b backend.Backend, cb config.Backend) error {
```

Comment:
Rolling in a unit refresh has been brought up a few times, it does make sense. For now, I just like allowing the user to use whatever units they want along with documenation, and having a little less machinery in deisctl.